Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -1,16 +1,65 @@
 #!/usr/bin/env python3
 
+import subprocess
 import json
+import time
 from az import *
+from helios import *
 import os
 
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 f = open('../cohesity.json',)
 data = json.load(f)
 resource_group = data['resource_group']
 workspace_name = data['workspace_name']
-playbook_name = data['playbook_name']
 apiKey = data['apiKey']
```

Comment:
s/apiKey/api_key

Please remove all such references in other files to be consistent.